### PR TITLE
update whitepaper to reflect catapult-server version 9.6.3

### PR DESCRIPTION
**Pull request by Jaguar0625**

### CODE REVIEW

File: whitepaper/bibliography.bib [hunk 1]
```diff
@@ -71,3 +71,19 @@ @Misc{Nxt2013
...
+  publisher            = {Internet Engineering Task Force},
+  note                 = {Work in Progress},
+  url                  = {https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-vrf-07},
+  author               = {Sharon Goldberg and Leonid Reyzin and Dimitrios Papadopoulos and Jan Včelák},
+  title                = {{Verifiable Random Functions (VRFs)}},
+  pagetotal            = 39,
+  year                 = 2020,
+  month                = jun,
+  day                  = 18,
+  abstract             = {A Verifiable Random Function (VRF) is the public-key version of a keyed cryptographic hash. Only the holder of the private key can compute the hash, but anyone with public key can verify the correctness of the hash. VRFs are useful for preventing enumeration of hash-based data structures. This document specifies several VRF constructions that are secure in the cryptographic random oracle model. One VRF uses RSA and the other VRF uses Eliptic Curves (EC).},
```
Comment (dgarcia360) [inline whitepaper/bibliography.bib]: Eliptic -> Elliptic

File: whitepaper/chapters/blockchain.tex [hunk 2]
```diff
@@ -152,16 +153,28 @@ \subsection{Block Generation}
...
 
+Making the generation hash dependent on a VRF makes it effectively random and unpredictable.
+This is important because it makes the identity of the next harvester unknown even for an adversary with perfect information.
+Until a block is pushed to the network, its generation hash is unknown to all but its harvester.
+Accordingly, until the best block is pushed to the network, since each generation hash is dependent on the preceding generation hash, the
+input into the generation hash calculation for the next block is unknown until that point.
+
+A VRF public key needs to be registered on the blockchain before it can be used to harvest a block.
+This preregistration requirement prevents an adversary from choosing an arbitrary VRF public key that can maximize its hit on the next block.
+As a result, the adversary is unable to "mine" blocks in an attempt find those with better hits.
```
Comment (dgarcia360) [inline whitepaper/chapters/blockchain.tex]: "mine" -> harvest, produce?

File: whitepaper/chapters/blockchain.tex [hunk 3]
```diff
@@ -152,16 +153,28 @@ \subsection{Block Generation}
...
 \index{generation hash}
 \index{block!generation hash}
 
-The generation hash of a block is derived from the previous block generation hash and the public key of the current block harvester:
+The generation hash of a block is derived from the previous block generation hash and the VRF proof included in the block:
 \begin{align*}
 	\tag{generation hash}
 	\mathfunc{gh}(1) = \: & \nemsetting{network}{generationHash} \\
-	\mathfunc{gh}(N) = \: & \hf(\mathfunc{gh}(N - 1), \mathname{public key of account})
+	\mathfunc{gh}(N) = \: & \mathfunc{verify\_vrf\_proof}(\mathfunc{proof}(block(N)), \mathfunc{gh}(N - 1), \mathname{vrf public key of account})
```
Comment (dgarcia360) [inline whitepaper/chapters/blockchain.tex]: vrf -> VRF

File: whitepaper/chapters/blocks.tex [hunk 4]
```diff
@@ -77,6 +80,10 @@ \subsection{Block Fields}
 \field{Difficulty} determines how hard is to harvest a new block, based on previous blocks.
 Difficulty is described in detail in \nemref{sec:blockchain:difficulty}.
 
+\field{GenerationHashProof} is the VRF proof generated with the block harvester's vrf private key.
```
Comment (dgarcia360) [inline whitepaper/chapters/blocks.tex]: vrf -> VRF

File: whitepaper/chapters/reputation.tex [hunk 5]
```diff
@@ -133,21 +133,22 @@ \subsection{Node Banning}
...
 			\hline
 		\end{tabular}
 	}{Banning Rules}
 \end{figure}
+\footnotetext{All active connections associated with the misbehaving node are closed immediately, not just the connection triggering the violation.}
 
 \subsubsection*{Consecutive Interaction Failures}
 If interactions with the same node fail for too many consecutive times due to networking or stateful failures, it is better to suspend all interactions with that node for some time, hoping the node will behave better in the future.
-The number of consecutive interaction failures before the node gets banned can be configured.
+The number of consecutive interaction failures before the node gets banned as a interaction partner can be configured.
```
Comment (dgarcia360) [inline whitepaper/chapters/reputation.tex]: a interaction -> an interaction

File: whitepaper/header/title.tex [hunk 6]
```diff
@@ -12,7 +12,7 @@
 	\fontsize{24pt}{25pt}\textit{Technical Reference}
 	\\
 	\vspace{20mm}
-	\fontsize{12pt}{14pt}{Version 1.0}
+	\fontsize{12pt}{14pt}{Version 0.9.6.3}
```
Comment (dgarcia360) [inline whitepaper/header/title.tex:15]: Consider changing the title "NEM Symbol" to "Symbol from NEM"

**Review by dgarcia360 — CHANGES_REQUESTED** [review]

**Review by dgarcia360 — DISMISSED** [review]

**Review by Wayonb — DISMISSED** [review]

**Pull request merged**